publicgen/reflect.go

```diff
@@ -0,0 +1,80 @@
+package publicgen
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+	"unicode"
+)
+
+var typeStruct = map[string]string{
```

Comment:
I don't really like the name `typeStruct`. It seems that you use this to map type to the aliased type. Maybe something like `aliasedTypeMap` is better?